include/ebpf_windows.h

```diff
@@ -70,9 +70,30 @@ typedef enum _ebpf_helper_function
 
 #define EBPF_MAX_GENERAL_HELPER_FUNCTION 0xFFFF
 
-#define EBPF_ATTACH_CLIENT_DATA_CURRENT_VERSION 1
-#define EBPF_PROGRAM_INFORMATION_CLIENT_DATA_CURRENT_VERSION 1
+// Warning: Version 1 of the eBPF extension data structures is being deprecated.
```

Comment:
This comment says "is being deprecated" implying now, whereas the other file implies it's not yet but will be in the future.

Reply:
Updated to make it clear that depreciation is in the future.